SPDX-FileCopyrightText: 2025 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: Artistic-1.0

The /r specifier is not supported in perl 5.14. Thankfully, it is
not necessary here.

diff --git perl-5.15.7/regen/mk_PL_charclass.pl perl-5.15.7/regen/mk_PL_charclass.pl
index a2f837fefc..f777ffe74d 100644
--- perl-5.15.7/regen/mk_PL_charclass.pl
+++ perl-5.15.7/regen/mk_PL_charclass.pl
@@ -64,7 +64,7 @@ while (<$fh>) {
     # Lines look like (without the initial '#'
     #0130; F; 0069 0307; # LATIN CAPITAL LETTER I WITH DOT ABOVE
     # Get rid of comments, ignore blank or comment-only lines
-    my $line = $_ =~ s/ (?: \s* \# .* )? $ //rx;
+    my $line = $_ =~ s/ (?: \s* \# .* )? $ //x;
     next unless length $line;
     my ($hex_from, $fold_type, @folded) = split /[\s;]+/, $line;
 
